app/views/showCollections.scala.html

```diff
@@ -1,16 +1,17 @@
-@(collections:Seq[model.TdrCollection])
```

Comment:
Should this be renamed as "showConsignments"

Reply:
done